tests/b036.py

```diff
@@ -0,0 +1,35 @@
+
+try:
+    pass
+except BaseException:  # bad
+    print("aaa")
+    pass
```

Comment:
Can we add and make this work easily? If not, no worries.

```suggestion
try:
    pass
except:  # bad
    print("aaa")
    pass
```

Reply:
Look like bare except already caught by both flake8 and bugbear:

```python
try:
    pass
except:
    pass
```
gives both:
- B001 Do not use bare `except:`, it also catches unexpected events like memory errors, interrupts, system exit, and so on.  Prefer `except Exception:`.  If you're sure what you're doing, be explicit and write `except BaseException:`.
- E722 do not use bare 'except'

Should this be consolidated with B001, rather than a new error?

Reply:
Hmm - that's a good question. I'd like that, but I wonder if that would upset people?

I vote yes, but lets see if any other contributors / maintainers weigh in here ...

Reply:
I guess keeping separate also gives you the flexibility to enable/disable either one? Maybe that's the safer option?

Reply:
Yeah ok, I like this now the more I think about it.